ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/hosts.yml

```diff
@@ -272,6 +272,8 @@ all:
                         mgmt_interface_fabric:
                         mgmt_interface_host:
                         mgmt_interface_platform:
+                        mgmt_interface_ipv6:
```

Comment:
please move these two up under line 13 so we only generate the minimum config.

Reply:
Fixed, thanks!